<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #3F0071;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background : #1C0C5B;
}
QPushButton#AddStudentButton {
background-color:#FF4C29;
color:white;
}

QPushButton#AddStudentButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#AddStudentButton:pressed {
    background-color: rgb(224, 0, 0);     
}
QPushButton#SessionButton {
background-color:#FF4C29;
color:white;
}

QPushButton#SessionButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#SessionButton:pressed {
    background-color: rgb(224, 0, 0);     
}
QPushButton#SearchStudentButton {
background-color:#FF4C29;
color:white;
}

QPushButton#SearchStudentButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#SearchStudentButton:pressed {
    background-color: rgb(224, 0, 0);     
}

QPushButton#AddTrainerButton {
background-color:#FF4C29;
color:white;
}

QPushButton#AddTrainerButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#AddTrainerButton:pressed {
    background-color: rgb(224, 0, 0);     
}

QPushButton#SearchTrainerButton {
background-color:#FF4C29;
color:white;
}

QPushButton#SearchTrainerButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#SearchTrainerButton:pressed {
    background-color: rgb(224, 0, 0);     
}

QPushButton#FeesButton {
background-color:#FF4C29;
color:white;
}

QPushButton#FeesButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#FeesButton:pressed {
    background-color: rgb(224, 0, 0);     
}

QPushButton#ExamButton {
background-color:#FF4C29;
color:white;
}

QPushButton#ExamButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#ExamButton:pressed {
    background-color: rgb(224, 0, 0);     
}
QPushButton#ExitButton {
background-color:#FF4C29;
color:white;
}

QPushButton#ExitButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#ExitButton:pressed {
    background-color: rgb(224, 0, 0);     
}
background : black ;</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>851</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF4C29;
background:#1C0C5B</string>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="AddStudentButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchStudentButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddTrainerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Trainer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchTrainerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search Trainer</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#1C0C5B</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="FeesButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExamButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exam</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExitButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SessionButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Session</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>-1</y>
      <width>841</width>
      <height>861</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : #3F0071;</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>15</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF4C29;</string>
     </property>
     <property name="text">
      <string>DASHBOARD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>79</y>
       <width>811</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#150050;
border-radius:30px;</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>19</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FF4C29;</string>
      </property>
      <property name="text">
       <string>No of Stuents Joined</string>
      </property>
     </widget>
     <widget class="QLabel" name="StudentNumber">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>811</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#150050;
border-radius:30px;</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>19</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#FF4C29;</string>
      </property>
      <property name="text">
       <string>No of Trainer Assigned</string>
      </property>
     </widget>
     <widget class="QLabel" name="TrainerNumber">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>145</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
